### ISSUE TITLE
AppLinks not working for flutter web

### ISSUE THREAD

**Issue by Faiza-Farooqui** [llfbandit/app_links#71]
**AppLinks not working for flutter web**

this package doesn't help us open flutter web page, like if i generated link to open any particular product from flutter web app it doesn't leads us to that particular page

**Comment by llfbandit** [comment]
The package does nothing by itself, when retrieving the link, it is up to you to navigate to the proper page if this is what you want.
Deeplinking does not mean routing (while this is often the case).

On web, the package gives you the current URL called.

**Comment by Faiza-Farooqui** [comment]
can you please tell me that if I have one flutter app, run on all platforms
(android, ios, flutter web), does it open the web or not? Have you tested
that on your flutter web?

**Comment by kayumarie** [comment]
Same here, I have an app running on iOS, Android. And crashing on web:

Error
```
UnimplementedError: allUriLinkStream not implemented on the current platform.
```

Stacktrace
```
dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/errors.dart 288:49  throw_
packages/app_links/src/app_links_platform_interface.dart 83:39                get allUriLinkStream
packages/app_links/src/app_links.dart 46:38                                   get allUriLinkStream
```

**Comment by HRahimy** [comment]
> Same here, I have an app running on iOS, Android. And crashing on web:
> 
> Error
> 
> ```
> UnimplementedError: allUriLinkStream not implemented on the current platform.
> ```
> 
> Stacktrace
> 
> ```
> dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/errors.dart 288:49  throw_
> packages/app_links/src/app_links_platform_interface.dart 83:39                get allUriLinkStream
> packages/app_links/src/app_links.dart 46:38                                   get allUriLinkStream
> ```

Getting the same error here as well. Running on flutter web.

Seems incompatible with web right now.

**Comment by llfbandit** [comment]
For those who have platform `UnimplementedError`. Please refresh your projects to ensure that the package is registered for web platform.

**Comment by llfbandit** [comment]
Sorry, I digged into the code tonight and saw that those methods where missing.
Fixed in v3.4.4.
